app/context-menus.js

```diff
@@ -143,6 +155,83 @@ function attachContextMenus ({ window, createWindow }) {
     ]
   }
 
+  function pageGroup (wc) {
+    return [
+      new MenuItem({
+        label: 'Create shortcut',
+        click: async () => {
+          const outputPath = (await dialog.showOpenDialog({
+            properties: ['openDirectory']
+          })).filePaths[0]
+          console.log(outputPath)
+          const appPath = process.argv[0]
+
+          const URL = wc.getURL()
```

Comment:
This should probably be a lowercase `url` since `URL` is also a global

Reply:
Good point